### DISCUSSION TITLE
Error websocket mqtt in rabbitmq

### DISCUSSION THREAD

**Discussion by ruem2802** [rabbitmq/rabbitmq-server#6959] [Other]
i use rabbitmq version: 3.11.7 but i can't connect http/web-mqtt in port: 15675, i use python(paho libary):
terminal have console: paho.mqtt.client.WebsocketConnectionError: WebSocket handshake error. help me. Thank
![image](https://user-images.githubusercontent.com/65638123/213500048-e5894f67-2e32-4087-8a20-6f7510225feb.png)
![image](https://user-images.githubusercontent.com/65638123/213500103-788e43b8-1646-41c4-9ff0-ee9369380164.png)

**Comment by michaelklishin** [comment]
We cannot suggest much with the amount of information provided. Consider putting together an executable example that would at least demonstrate the code and provide access to server logs.

**Comment by lukebakken** [comment]
It looks like you are using the [Paho Python MQTT library](https://pypi.org/project/paho-mqtt/) but are trying to connect to the MQTT WebSocket port. That library is **not compatible** with WebSockets as far as I know. You should use a WebSocket compatible library - https://www.rabbitmq.com/web-mqtt.html#usage
